main.py

```diff
@@ -52,10 +52,12 @@ async def on_ready():
 
 @bot.event
 async def on_message(message):
-    try:
-        await bot.process_commands(message)
-    except commands.errors.CommandNotFound:
-        await message.channel.send(f'I couldn\'t find that command, please use {PREFIX}help for a list of commands.')
+    await bot.process_commands(message)
```

Comment:
Am I going crazy or is this the default behaviour if you didn't have an on_message function?

Reply:
yeah you're right I just totally forgot, changed